src/custom/pages/Profile/ens.ts

```diff
@@ -0,0 +1,54 @@
+import { SupportedChainId } from 'constants/chains'
+import { ClientError, gql, GraphQLClient } from 'graphql-request'
+import { EnsNamesQuery } from 'state/data/generated'
+
+// List of supported subgraphs. Note that the app currently only support one active subgraph at a time
```

Comment:
What does this mean? That we can't have more than 1 ENS subgraph or any kind of subgraph?

Reply:
I removed this comment and also added the rinkeby subgraph that was missing